fl4health/checkpointing/server_module.py

```diff
@@ -0,0 +1,580 @@
+from logging import INFO
+from typing import Any, Dict, Sequence, Union
+
+import torch.nn as nn
+from flwr.common import Parameters
+from flwr.common.logger import log
+from flwr.common.parameter import parameters_to_ndarrays
+from flwr.common.typing import Scalar
+
+from fl4health.checkpointing.checkpointer import PerRoundStateCheckpointer, TorchModuleCheckpointer
+from fl4health.checkpointing.opacus_checkpointer import OpacusCheckpointer
+from fl4health.parameter_exchange.packing_exchanger import FullParameterExchangerWithPacking
+from fl4health.parameter_exchange.parameter_exchanger_base import ExchangerType
+from fl4health.parameter_exchange.parameter_packer import (
+    ParameterPackerAdaptiveConstraint,
+    ParameterPackerWithClippingBit,
+    ParameterPackerWithControlVariates,
+    ParameterPackerWithLayerNames,
+    SparseCooParameterPacker,
+)
+
+CheckpointModuleInput = Union[TorchModuleCheckpointer, Sequence[TorchModuleCheckpointer]] | None
```

Comment:
Same comment here with the typing, is it better to be more explicit by removing this type definition?